packages/runtimes/hmr/src/loaders/hmr-runtime.js

```diff
@@ -120,9 +130,15 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
       if (handled) {
         console.clear();
 
-        assets.forEach(function (asset) {
-          hmrApply(module.bundle.root, asset);
-        });
+        // Dispatch custom event so other runtimes (e.g React Refresh) are aware.
+        if (
+          typeof window !== 'undefined' &&
+          typeof CustomEvent !== 'undefined'
+        ) {
+          window.dispatchEvent(new CustomEvent('parcelhmraccept'));
```

Comment:
Minor thing but you might instead use `parcel-hmr-accept` in styling convention with DOM events. Also since you're already using global state you could also a well-known symbol to `global` containing a `registerAcceptor` function or something to avoid polluting user code.

Reply:
hmm I think most DOM events actually don't use dashes? And the CustomEvent example on MDN also doesn't... https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent